plugins/woocommerce/src/Admin/PluginsHelper.php

```diff
@@ -317,6 +318,13 @@ public static function install_plugins( $plugins, ?PluginsInstallLogger $logger
 				continue;
 			}
 
+			/**
+			 * Action triggered before a plugin is installed.
+			 *
+			 * @since 9.8
+			 */
+			do_action( 'woocommerce_plugins_install_before', $slug, $source );
```

Comment:
Since we're adding the option only when installing a plugin, we will not support pre-installed plugins for certain hosts. Do we need to consider on activate? cc @adrianduffell